src/components/sprite-selector/sprite-list.jsx

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+import DragConstants from '../../lib/drag-constants';
+
+import Box from '../box/box.jsx';
+import SpriteSelectorItem from '../../containers/sprite-selector-item.jsx';
+import SortableHOC from '../../lib/sortable-hoc.jsx';
+import SortableAsset from '../asset-panel/sortable-asset.jsx';
+
+import styles from './sprite-selector.css';
+
+const SpriteList = function (props) {
+    const {
+        editingTarget,
+        draggingIndex,
+        draggingType,
+        hoveredTarget,
+        onDeleteSprite,
+        onDuplicateSprite,
+        onSelectSprite,
+        onAddSortable,
+        onRemoveSortable,
+        ordering,
+        raised,
+        selectedId,
+        items
+    } = props;
+
+    const isSpriteDrag = draggingType === DragConstants.SPRITE;
+
+    return (
+        <Box className={styles.itemsWrapper}>
+            {items.map((sprite, index) => (
+                <SortableAsset
+                    className={classNames(styles.itemWrapper, {
+                        [styles.placeholder]: isSpriteDrag && index === draggingIndex})}
+                    index={isSpriteDrag ? ordering.indexOf(index) : index}
+                    key={sprite.name}
+                    onAddSortable={onAddSortable}
+                    onRemoveSortable={onRemoveSortable}
+                >
+                    <SpriteSelectorItem
+                        assetId={sprite.costume && sprite.costume.assetId}
+                        className={hoveredTarget.sprite === sprite.id &&
+                                sprite.id !== editingTarget &&
+                                hoveredTarget.receivedBlocks ?
+                            classNames(styles.sprite, styles.receivedBlocks) :
+                            raised && sprite.id !== editingTarget ?
+                                classNames(styles.sprite, styles.raised) : styles.sprite}
```

Comment:
Oh totally agree, I just moved this directly from what already existed and was hoping to avoid refactoring it, but you are right this is a good chance to clean it up.